bin/metrics-disk-capacity.rb

```diff
@@ -68,7 +68,7 @@ def run
 
         timestamp = Time.now.to_i
         if fs =~ /\/dev/
-          fs = fs.tr('/dev/', '')
+          fs = fs.sub('/dev/', '')
```

Comment:
I had not used `sub` (very familiar with `gsub`) before and had to compare them from what I can tell the difference is that  it replaces only the first occurrence as opposed to all occurrences with `gsub` and `tr`. Is that right?

Reply:
I have updated the changelog to reflect this as I think it's pretty nuanced and not obvious without inspecting the documentation for each side by side.

Reply:
`tr` translates every occurrence of the characters, `sub` substitutes the first occurrence of the specified string, whereas `gsub` substitutes all occurrences of the string.